src/package_io/input_parser.star

```diff
@@ -36,7 +36,7 @@ DEFAULT_CL_IMAGES_MINIMAL = {
 
 DEFAULT_VC_IMAGES = {
     "lighthouse": "ethpandaops/lighthouse:stable",
-    "lodestar": "chainsafe/lodestar:latest",
+    "lodestar": "ethpandaops/lodestar:unstable",
```

Comment:
I guess this can be reverted since we have it now in our latest image

Reply:
Fixed.